docs/microbit_micropython_api.rst

```diff
@@ -102,9 +136,26 @@ Each of these pins are instances of the ``MicroBitPin`` class, which offers the
     # sets the period of the PWM output of the pin in microseconds
     # (see https://en.wikipedia.org/wiki/Pulse-width_modulation)
     pin.set_analog_period_microseconds(int)
-    # returns boolean
+    # returns boolean for touch pins; 0,1 and 2
     pin.is_touched()
 
+Except in the case of the pins marked **V2**, which offers the following API::
+    
+pin_logo::
+
+    # returns boolean for logo touch pin
+    pin_logo.is_touched()
+
+pin_speaker::
+    
+As above `MicroBitPin` class, but does not include pin.is_touched() and
+includes:
```

Comment:
Will this render correctly without the double colon? `::`

Reply:
![image](https://user-images.githubusercontent.com/29712657/100445413-f8b03780-30a4-11eb-9874-98b78f52615d.png)

Reply:
Fixed